scarb/src/core/package/id.rs

```diff
@@ -9,6 +9,7 @@ use crate::core::source::SourceId;
 use crate::core::PackageName;
 use crate::internal::static_hash_cache::StaticHashCache;
 use crate::internal::to_version::ToVersion;
+const TEST_PACKAGE_PREFIX: &str = "___test_package_prefix___";
```

Comment:
Out of curiosity, how does Cargo prefix these? I'm afraid we'll end up showing this prefix to users for example in error messages, so maybe we should come up with something more "readable". Say `X_unittests`